Front camera switch implementation (#92)
spoonconsulting/cordova-plugin-simple-camera-preview: README.md

### switchCameraTo(option, successCallback, errorCallback)

Switch camera between wide or auto and set the camera direction (front or back) dynamically.

> The variable lens and direction can take two values:

Reviewer: ```suggestion
The options variable can take the following keys:
```
rename 'option' to 'options' i think it's a typo in the readme

#### Available options:

> - **Lens:**

Reviewer: ```suggestion
- **lens:**
```
should match case being used in the method

  - `"wide"` – Use a wide-angle lens.
  - `"auto"` – Automatically select the best available lens.

> - **Direction:**

Reviewer: ```suggestion
- **direction:**
```

same here
